cluster/gce/config-test.sh

```diff
@@ -51,6 +51,7 @@ TERMINATED_POD_GC_THRESHOLD=${TERMINATED_POD_GC_THRESHOLD:-100}
 # Optional: enable v1beta1 related features
 ENABLE_DEPLOYMENTS="${KUBE_ENABLE_DEPLOYMENTS:-true}"
 ENABLE_DAEMONSETS="${KUBE_ENABLE_DAEMONSETS:-true}"
+ENABLE_REPLICASETS="${KUBE_ENABLE_REPLICASETS:-true}"
```

Comment:
the integration of support for replicasets into the cluster scripts seems incomplete when compared to something like daemonsets:
```sh
$ find . -type f -exec grep -H -e DAEMONSET \{\} \;
./cluster/aws/config-default.sh:ENABLE_DAEMONSETS="${KUBE_ENABLE_DAEMONSETS:-}"
./cluster/aws/config-test.sh:ENABLE_DAEMONSETS="${KUBE_ENABLE_DAEMONSETS:-true}"
./cluster/common.sh:  if [[ "${ENABLE_DAEMONSETS}" == "true" ]]; then
./cluster/gce/config-default.sh:ENABLE_DAEMONSETS="${KUBE_ENABLE_DAEMONSETS:-}"
./cluster/gce/config-test.sh:ENABLE_DAEMONSETS="${KUBE_ENABLE_DAEMONSETS:-true}"
./cluster/kubemark/config-default.sh:ENABLE_DAEMONSETS="${KUBE_ENABLE_DAEMONSETS:-true}"
./hack/jenkins/e2e.sh:    KUBE_ENABLE_DAEMONSETS=true
./hack/jenkins/e2e.sh:    : ${KUBE_ENABLE_DAEMONSETS:=true}
./hack/jenkins/e2e.sh:    : ${KUBE_ENABLE_DAEMONSETS:=true}
./docs/admin/daemons.md:achieved by exporting KUBE_ENABLE_DAEMONSETS=true before running kube-up.sh script
```